test/attr/attr_inlinable.swift

```diff
@@ -248,3 +248,11 @@ public struct KeypathStruct {
     // expected-error@-1 {{property 'x' is internal and cannot be referenced from an '@inlinable' function}}
   }
 }
+
+public struct HasInternalSetProperty {
+  public internal(set) var x: Int // expected-note {{setter for 'x' is not '@usableFromInline' or public}}
+
+  @inlinable public mutating func setsX() {
+    x = 10 // expected-error {{setter for 'x' is internal and cannot be referenced from an '@inlinable' function}}
```

Comment:
Oops, wait, we can't make this an error in 4.2 mode.

Reply:
@slavapestov Any preference? I think we've been leaning towards always putting the latest behavior in the "normal" file and older behavior in "Compatibility".

Reply:
Went ahead and just did that